landoapi/spec/swagger.yml

```diff
@@ -121,6 +121,45 @@ paths:
           schema:
             allOf:
               - $ref: '#/definitions/Error'
+  /landings/{landing_id}/update:
+    post:
+      operationId: landoapi.api.landings.update
+      description: |
+        Sends request to transplant service and responds just the status code
+      parameters:
+        - name: data
+          in: body
+          description: |
+            Retrieve status of the landing job
+          required: true
+          schema:
```

Comment:
See https://docs.google.com/document/d/1cRIdpVqYcn0Mle7vyjJePNw5TcFoZ20N42dLVy5xVEk/edit# for the remaining data.